scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -1259,8 +1259,19 @@
                                      (log/error "Unable to decline offer; no current driver"))))))
       (build)))
 
+(defn persist-mea-culpa-failure-limit!
+  "The Datomic transactor needs to be able to access this part of the
+  configuration, so on cook startup we transact the configured value into Datomic."
+  [conn limit]
+  (when limit
+    @(d/transact conn [{:db/id :scheduler/config
+                        :scheduler.config/mea-culpa-failure-limit (biginteger limit)}])))
```

Comment:
Goooooood question.   It should probably just be a long in the db schema rather than a bigint, and then this type conversion wouldn't be necessary.